dockerHashing/src/main/scala/cromwell/docker/DockerInfoActor.scala

```diff
@@ -223,22 +224,17 @@ object DockerInfoActor {
   def props(dockerRegistryFlows: Seq[DockerRegistry],
             queueBufferSize: Int = 100,
             cacheEntryTTL: FiniteDuration,
-            cacheSize: Long) = {
+            cacheSize: Long): Props = {
     Props(new DockerInfoActor(dockerRegistryFlows, queueBufferSize, cacheEntryTTL, cacheSize)).withDispatcher(Dispatcher.IoDispatcher)
   }
 
   def remoteRegistriesFromConfig(config: Config): List[DockerRegistry] = {
     import cats.syntax.traverse._
 
-    val googleConstructor = { c: DockerRegistryConfig =>
-      c.copy(throttle = c.throttle.orElse(DockerConfiguration.instance.deprecatedGcrApiQueriesPer100Seconds))
```

Comment:
A failed attempt to pass in `deprecatedGcrApiQueriesPer100Seconds`: the result of the `.copy` is thrown away and not passed into the constructed `GoogleRegistry`.

Reply:
Bravo!